qermit/postselection/postselect_manager.py

```diff
@@ -0,0 +1,107 @@
+from collections import Counter
+from pytket.backends.backendresult import BackendResult
+from pytket.utils.outcomearray import OutcomeArray
+from pytket.circuit import Bit
+from typing import List, Tuple, Dict
+
+
+class PostselectMgr:
+    """Class for tracking and applying post selection to results.
+    Includes other methods to analyse the results after post selection.
+    """
+
+    def __init__(
+        self,
+        compute_cbits: List[Bit],
+        postselect_cbits: List[Bit],
+    ):
+        """Initialisation method.
+
+        :param compute_cbits: Bits in the circuit which are not affected
+            by post selection.
+        :type compute_cbits: List[Bit]
+        :param postselect_cbits: Bits on which the post selection is based.
+        :type postselect_cbits: List[Bit]
+        :raises Exception: Raised if a bit is in both compute_cbits
+            and postselect_cbits.
+        """
+
+        intersect = set(compute_cbits).intersection(set(postselect_cbits))
+        if intersect:
+            raise Exception(
+                f"{intersect} are post select and compute qubits. "
+                + "They cannot be both."
+            )
+
+        self.compute_cbits = compute_cbits
```

Comment:
I'd add types for the class attributes, so self.postselect_cbits: List[Bit] = postselect_cbits, etc.